CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+# [0.7.2] - 2020-07-27
```

Comment:
Need to find some time to explore GitHub releases and get this generated automatically (as we do in C++/WinRT).